### PR TITLE
add file structure

### PR DESCRIPTION
**Pull request by mariamisa**
create file structure
relates #31

### CODE REVIEW

File: client/src/App/style.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+}
+
+@keyframes App-logo-spin {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
```
Comment (Osama-you) [inline client/src/App/style.css]: I think these selectors are not needed

File: client/src/index.css [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+code {
+  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
+    monospace;
+}
```
Comment (Osama-you) [inline client/src/index.css]: Also here the same.

File: client/public/index.html [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
+    />
```
Comment (Osama-you) [inline client/public/index.html]: Add a short description of our app.

File: client/public/index.html [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+
+    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+
+    -->
```
Comment (Osama-you) [inline client/public/index.html]: We don't have a manifest.json file

**Review by zeinjendeya — APPROVED** [review]

**Review by Osama-you — APPROVED** [review]

**Review by mAbadsa — APPROVED** [review]

**Pull request merged**